lib/tools/adb-emu-commands.js

```diff
@@ -18,29 +18,37 @@ emuMethods.isEmulatorConnected = async function (udid) {
   return false;
 };
 
+emuMethods.checkEmulatorConnected = async function (udid) {
```

Comment:
@jlipps I clean up and refactor the code a bit, let me know what you think
https://github.com/vrunoa/appium-adb/blob/add-emu-rotate/lib/tools/adb-emu-commands.js#L25-L35